change mapping vs form policy order of prediction (#5448)
RasaHQ/rasa: changelog/5205.bugfix.rst

> Fixed the bug when FormPolicy was overwriting MappingPolicy prediction (e.g. /restart).

Reviewer: ```suggestion
Fixed the bug when FormPolicy was overwriting MappingPolicy prediction (e.g. ``/restart``).
```

> Priorities for MappingPolicy and FormPolicy are no longer linear:

Reviewer: A link for `FormPolicy` would also be great
```suggestion
Priorities for :ref:`mapping-policy:` and FormPolicy are no longer linear:
```

> FormPolicy priority is 5, but its prediction is ignored if MappingPolicy is used for prediction.

Reviewer: ```suggestion
FormPolicy priority is 5, but its prediction is ignored if ``MappingPolicy`` is used for prediction.
```
